shared/src/balancer/pool_cache.rs

```diff
@@ -3,7 +3,7 @@ use crate::balancer::pool_storage::RegisteredWeightedPool;
 use crate::pool_fetching::{handle_contract_error, MAX_BATCH_SIZE};
 use crate::{
     balancer::pool_storage::WeightedPool,
-    recent_block_cache::{Block, CacheFetching, CacheKey, CacheMetrics, RecentBlockCache},
+    recent_block_cache::{Block, CacheFetching, CacheKey, RecentBlockCache},
```

Comment:
nit: This is an unnecessary change - it looks like the `use` statement was just moved to a different line.

Reply:
Yea, I just went over all the import statements and consolidated them. So there are a few more changed imports but now they are all in the most compact form throughout the module.